Added a layout section to hide and show panels.

tinnakaren opened the pull request:
This is an initial request to get some basic layout options in. Should be improved to allow resizable panels etc as well.

Commit:
Added a layout section to hide and show panels.

Commit:
Removed obsolete code.

tinnakaren commented:
This is an initial request to get some basic layout options in. Should be improved to allow resizable panels etc as well.

eisisig commented:
I like where this is going. But I don't like where it is and how it looks. BUT then again nothing has been designed so... But maybe a settings menu/icon in the side bar which will hold these things? Can be done in different branch

But one thing I'd like to see for these "settings" is to store the state in localStorage? So you don't have to toggle these things over and over? Should be pretty easy

Commit:
Merge branch 'master' of github.com:eisisig/react-isolate into feature/layout-changes

Commit:
Moved the Layout panel and renamed to Views.

tinnakaren commented:
@eisisig I changed the position of where the layout options were and renamed to Views. What do you think of this? It doesn't take up much space now and I feel like it's in the most natural place it should be, since it's meant to affect the views of the panels below.

Commit:
Removed obsolete code from App.js
